src/components/proposal/ProposalContainer.tsx

```diff
@@ -77,11 +77,18 @@ const proposalReducer = (
 const isProposalSubmitted = (proposal: Pick<Proposal, 'submitted'>) =>
   proposal.submitted;
 
-function isReadOnly(proposal: ProposalSubsetSubmission) {
+function GetProposalStatus(proposal: ProposalSubsetSubmission) {
+  if (proposal.status != null) {
+    return proposal.status?.shortCode.toString();
+  } else {
+    return 'Proposal Status is null';
+  }
+}
+
+function isReadOnly(proposalToCheck: ProposalSubsetSubmission) {
   if (
-    proposal.status != null &&
-    (proposal.status.shortCode.toString() === 'DRAFT' ||
-      proposal.status.shortCode.toString() === 'EDITABLE_SUBMITTED')
+    !proposalToCheck.submitted ||
+    GetProposalStatus(proposalToCheck) === 'EDITABLE_SUBMITTED'
```

Comment:
After running the tests locally it seems like this is the line that fails the final test. By adding 'DRAFT' back as an option in here it seems to fix it. However the logic does seem correct, it is read-only if it hasn't been submitted or if it is editable after submission.

Reply:
After some investigation I think this read-only function is just for viewing a submitted proposal not for editing proposal. I think that proposals that are editable should not get to this code at least while the call is active. So I think this change needs to be reverted but would like some clarification if there is someone some more familiar with this section of the frontend.

Reply:
I looked into it. It looks like the read-only state is set properly, however, for the ProposalSummary whole view was disabled, and the test was trying to click the view RichText content button but was unable to because all mouse pointers were disabled. I made a PR to address the problem. 
https://github.com/UserOfficeProject/user-office-frontend/pull/421

Reply:
Cool thanks for the explanation, I was a bit confused by this.

Reply:
When I ran this test locally, the proposal used in the test has a submitted value of true, but it somehow has a status of 1 (draft). Which would explain why adding the OR draft check back in does get the test passing again. 

I think commit 02002c6 is still necessary, as this read-only logic also seems to be used in the questionary (e.g. when you create a new proposal, whether the fields are disabled or not). So if it's in editable submitted status read only should be false to enable users to edit proposals in this state (which there aren't currently any tests for).